src/library/Uno.Cupertino/Styles/Controls/Slider.xaml

```diff
@@ -21,7 +21,7 @@
 	<StaticResource x:Key="CupertinoSliderInlineTickBarFill"
 					ResourceKey="CupertinoBlueColor" />
 
-	<!-- Cupertino Slider Thumb Style -->
+	<!--  Cupertino Slider Thumb Style  -->
```

Comment:
unnecessary change, you can change that your setting: Xaml Styler > Misc > Comment Padding > 1 (from 2)